cfep-XX.md

```diff
@@ -0,0 +1,61 @@
+
+<table>
+<tr><td> Title </td><td> Guidelines for (Re)Naming Packages </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td>
+    Jaime Rodríguez-Guerra &lt;jaime.rogue@gmail.com&gt;, Matthew R. Becker &lt;becker.mr@gmail.com&gt
+</td></tr>
+<tr><td> Created </td><td> Oct 26, 2025 </td></tr>
+<tr><td> Updated </td><td> Oct 26, 2025 </td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda-forge/admin-requests/pull/1711 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CFEP describes the principles informing how and when to (re)name a package in conda-forge.
+Two cases are described depending on the status of the target name:
+
+- Unclaimed names: A new package or a package already present in conda-forge wants to use a new name currently unclaimed.
+- Names in use: A package, already present or about to be submitted, wishes to use a name already in use.
+
+## Specification
+
+This CFEP follows [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) in its use of certain keywords (e.g., MUST).
+
+### Unclaimed Names
+
+#### New Feedstocks & Name Squatting
+
+- Unclaimed package names are typically available on a first-come-first-served basis and can be claimed by submitting a recipe for a package via `staged-recipes` (or equivalent).
+- The core team reserves the right to prevent the use of certain names for any reason. Typical reasons the core team may prevent the use of a certain name include, but are not limited to, typo squating, ambiguity, violations of the code of conduct, conflicts with important infrastructure, etc.
```

Comment:
squating -> squatting

Reply:
```suggestion
- The core team reserves the right to prevent the use of certain names for any reason. Typical reasons the core team may prevent the use of a certain name include, but are not limited to, typo squatting, ambiguity, violations of the code of conduct, conflicts with important infrastructure, etc.
```